custom_components/smartthinq_sensors/sensor.py

```diff
@@ -297,6 +311,68 @@
     DeviceType.WASHER,
 ]
 
+OVEN_SENSORS = {
+    DEFAULT_SENSOR: {
+        ATTR_MEASUREMENT_NAME: "Default",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FN: lambda x: x._power_state,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_LEFT_FRONT_STATE: {
+        ATTR_MEASUREMENT_NAME: "Cooktop Left Front",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FEAT: FEAT_COOKTOP_LEFT_FRONT_STATE,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_LEFT_REAR_STATE: {
+        ATTR_MEASUREMENT_NAME: "Cooktop Left Rear",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FEAT: FEAT_COOKTOP_LEFT_REAR_STATE,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_CENTER_STATE: {
+        ATTR_MEASUREMENT_NAME: "Cooktop Center",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FEAT: FEAT_COOKTOP_CENTER_STATE,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_RIGHT_FRONT_STATE: {
+        ATTR_MEASUREMENT_NAME: "Cooktop Right Front",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FEAT: FEAT_COOKTOP_RIGHT_FRONT_STATE,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_RIGHT_REAR_STATE: {
+        ATTR_MEASUREMENT_NAME: "Cooktop Right Rear",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FEAT: FEAT_COOKTOP_RIGHT_REAR_STATE,
+        ATTR_ENABLED: True,
+    },
+    FEAT_OVEN_LOWER_STATE: {
+        ATTR_MEASUREMENT_NAME: "Lower",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FEAT: FEAT_OVEN_LOWER_STATE,
+        ATTR_ENABLED: True,
+    },
+    FEAT_OVEN_LOWER_TARGET_TEMP: {
+        ATTR_MEASUREMENT_NAME: "Lower Target Temperature",
```

Comment:
This could need `ATTR_DEVICE_CLASS: DEVICE_CLASS_TEMPERATURE` and related `ATTR_UNIT_FN`?

Reply:
Fixed